src/functions/archiveTechRecordStatus.ts

```diff
@@ -0,0 +1,26 @@
+import TechRecordsDAO from "../models/TechRecordsDAO";
+import TechRecordsService from "../services/TechRecordsService";
+import HTTPResponse from "../models/HTTPResponse";
+import ITechRecordWrapper from "../../@Types/ITechRecordWrapper";
+
+export async function archiveTechRecordStatus(event: any) {
+    const techRecordsService = new TechRecordsService(new TechRecordsDAO());
+
+    const systemNumber: string = event.pathParameters!.systemNumber;
+    const techRec = event.body ? event.body.techRecord : null;
```

Comment:
I think we should add a check if techRec is on the payload, otherwise if not passed it will fail

Reply:
We should throw an error if techRecord is not on the body